src/command/global/settings.ts

```diff
@@ -0,0 +1,163 @@
+import { SlashCommandBuilder } from "@discordjs/builders";
+import {
+    ButtonInteraction,
+    CommandInteraction,
+    MessageActionRow,
+    MessageButton,
+} from "discord.js";
+import {
+    NotificationSettingsRepository,
+    Toggles,
+} from "../../repository/NotficationSettingsRespository";
+import { getCustomRepository } from "typeorm";
+import { NotificationSettings } from "../../entity/NotificationSettings";
+
+export const data = new SlashCommandBuilder()
+    .setName("settings")
+    .setDescription("View and toggle settings for notifications");
+
+enum Color {
+    PRIMARY = "PRIMARY",
+    SECONDARY = "SECONDARY",
+}
+
+function getColorFromBoolean(bool: boolean) {
```

Comment:
specify the return type for clarity

Reply:
actually, why does this function exist in the first place? It's only used by getStyle and seems kind of redundant

Reply:
for modularity/readability. i was taught to always break down code into many smaller pieces. each does only one thing. (but js doesnt really work like that and its breaking my brain)